build/Dockerfile

```diff
@@ -2,8 +2,8 @@ FROM quay.io/operator-framework/ansible-operator:v0.9.0
 
 USER root
 RUN yum -y update && yum clean all
+RUN yum -y install python-boto3
```

Comment:
Add into the line above or add && yum clean all, please. Leaving the cache files makes the layer unnecessarily large.